src/data.js

```diff
@@ -117,25 +116,25 @@ Namespace.ensureNamespace = function(namespace, expires) {
 
     // Create parent namespaces
     return Namespace.ensureNamespace.call(
-      Class,
+      that,
       namespace,
       expires
     ).then(function() {
```

Comment:
If this was an arrow function, we wouldn't need `that`, right?

Reply:
I expect that's true. I just did a sed here. I can look into making it nicer shortly. I guess this can't get landed anytime soon anyway.